win32/GNUmakefile

```diff
@@ -1154,7 +1159,15 @@ ifeq ($(CCTYPE),)
 endif
 
 
-..\regcomp$(o) : ..\regnodes.h ..\regcharclass.h
+..\regcomp$(o) : ..\regcomp.h ..\regcomp_internal.h ..\regnodes.h ..\regcharclass.h
```

Comment:
This doesn't require any action on your part, but this screams "automate me" to me.

Reply:
Oh god, me too. I already did some similar automation for the re module. But automating it in the win32 makefiles seemed a touch ambitious for me right now. :-)